### PR TITLE
[TEST] update org map

### PR DESCRIPTION
**Pull request by bumblebee89**

### CODE REVIEW

**Review by ajinkyan83 — APPROVED** [review]

File: map_org_names.yaml
Comment (lukaszgryglicki) [inline map_org_names.yaml:2820]: Don't add another IBM account, see one line above - yesterday I've merged two IBM accounts.

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
See my comment cc @1010sachin

File: map_org_names.yaml
Comment (bumblebee89) [reply map_org_names.yaml]: @lukaszgryglicki done

**Comment by lukaszgryglicki** [comment]
I can still see the line
```
- ['^[[:space:]]*((account[[:space:]])?[[:space:]]*international[[:space:]]+business[[:space:]]+machines([[:space:]]+corporation)?([[:space:]]+\\(?ibm\\)?)?|ibm)[[:space:]]*$','International Business Machines Corporation']
```
being removed.

**Comment by bumblebee89** [comment]
Hi @lukaszgryglicki I have removed my additional IBM line and left the existing as you requested. Please when you get some time, drop a lgtm if ok and i merge. Thanks

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Comment by lukaszgryglicki** [comment]
1st run:
```
{"text":"Updated organization '1 & 1 Internet SE' -> '1&1 Telecommunication SE' on 2 enrollments, Deleted organization '1 & 1 Internet SE' (id=65712), Updated organization '1&1 Mail&Media Development & Technology GmbH' -> '1&1 Telecommunication SE' on 9 enrollments, Updated organization '1&1 Mail&Media Development & Technology GmbH' -> '1&1 Telecommunication SE' on 95 archived enrollments, Deleted organization '1&1 Mail&Media Development & Technology GmbH' (id=40898), Updated organization 'Adfinis SyGroup' -> 'Adfinis AG (f/k/a Adfinis SyGroup)' on 17 enrollments, Updated organization 'Adfinis SyGroup' -> 'Adfinis AG (f/k/a Adfinis SyGroup)' on 36 archived enrollments, Deleted organization 'Adfinis SyGroup' (id=74335), Updated organization 'Alcatel Lucent' -> 'ALE International SAS' on 7 enrollments, Deleted organization 'Alcatel Lucent' (id=55244), Updated organization 'Allegro' -> 'Allegro.pl sp. z o.o.' on 135 enrollments, Updated organization 'Allegro' -> 'Allegro.pl sp. z o.o.' on 735 archived enrollments, Deleted organization 'Allegro' (id=27369), Updated organization 'Amazon Inc' -> 'Amazon.com, Inc.' on 3 enrollments, Updated organization 'Amazon Inc' -> 'Amazon.com, Inc.' on 1 archived enrollments, Deleted organization 'Amazon Inc' (id=72404), Updated organization 'American Express' -> 'American Express Banking Corp.' on 99 enrollments, Updated organization 'American Express' -> 'American Express Banking Corp.' on 454 archived enrollments, Deleted organization 'American Express' (id=32208), Updated organization 'Apple' -> 'Apple Inc.' on 1534 enrollments, Updated organization 'Apple' -> 'Apple Inc.' on 3454 archived enrollments, Deleted organization 'Apple' (id=28097), Updated organization 'Bank Salad' -> 'Banksalad Co., Ltd.' on 12 enrollments, Updated organization 'Bank Salad' -> 'Banksalad Co., Ltd.' on 48 archived enrollments, Deleted organization 'Bank Salad' (id=59035), Added organization 'Blackwood Seven A/S' (id=78065), Updated organization 'Blackwood Seven' -> 'Blackwood Seven A/S' on 27 enrollments, Updated organization 'Blackwood Seven' -> 'Blackwood Seven A/S' on 57 archived enrollments, Deleted organization 'Blackwood Seven' (id=31253), Deleted organization 'CGI Inc.' (id=75340), Updated organization 'Helm.' -> 'Cloud Native Computing Foundation' on 5 enrollments, Updated organization 'Helm.' -> 'Cloud Native Computing Foundation' on 6 archived enrollments, Deleted organization 'Helm.' (id=43501), Updated organization 'Cloudera Inc.' -> 'Cloudera, Inc.' on 307 enrollments, Updated organization 'Cloudera Inc.' -> 'Cloudera, Inc.' on 901 archived enrollments, Deleted organization 'Cloudera Inc.' (id=56745), Updated organization 'Plaxo' -> 'Comcast Corporation' on 26 enrollments, Updated organization 'Plaxo' -> 'Comcast Corporation' on 33 archived enrollments, Deleted organization 'Plaxo' (id=28768), Updated organization 'ChartIQ' -> 'Cosaic, Inc.' on 19 enrollments, Updated organization 'ChartIQ' -> 'Cosaic, Inc.' on 104 archived enrollments, Deleted organization 'ChartIQ' (id=35445), Updated organization 'D2iQ, Inc.' -> 'D2iQ, Inc. (f/k/a Mesosphere)' on 936 enrollments, Updated organization 'D2iQ, Inc.' -> 'D2iQ, Inc. (f/k/a Mesosphere)' on 5231 archived enrollments, Deleted organization 'D2iQ, Inc.' (id=74181), Updated organization 'Devops Ltd.' -> 'Devops Ltd. (ProdOps)' on 3 enrollments, Deleted organization 'Devops Ltd.' (id=74442), Updated organization 'Electronic' -> 'Electronic Arts Inc.' on 57 enrollments, Updated organization 'Electronic' -> 'Electronic Arts Inc.' on 59 archived enrollments, Deleted organization 'Electronic' (id=29565), Updated organization 'Embark Studios' -> 'Embark Studios AB' on 1 enrollments, Updated organization 'Embark Studios' -> 'Embark Studios AB' on 2 archived enrollments, Deleted organization 'Embark Studios' (id=71471), Updated organization 'Telefonaktiebolaget LM Ericsson' -> 'Ericsson' on 1 enrollments, Deleted organization 'Telefonaktiebolaget LM Ericsson' (id=73952), Updated organization 'Flipkart' -> 'Flipkart Internet Private Limited' on 74 enrollments, Updated organization 'Flipkart' -> 'Flipkart Internet Private Limited' on 344 archived enrollments, Deleted organization 'Flipkart' (id=26396), Updated organization 'Deutsche Niederlassung der FRIDAY Insurance S.A.' -> 'FRIDAY Insurance SA' on 13 enrollments, Deleted organization 'Deutsche Niederlassung der FRIDAY Insurance S.A.' (id=56814), Updated organization 'Sentry.io' -> 'Functional Software, Inc. (Sentry)' on 1 enrollments, Deleted organization 'Sentry.io' (id=69843), Added organization 'GE Digital Holdings LLC' (id=78066), Updated organization 'GE Digital' -> 'GE Digital Holdings LLC' on 65 enrollments, Updated organization 'GE Digital' -> 'GE Digital Holdings LLC' on 148 archived enrollments, Deleted organization 'GE Digital' (id=32891), Updated organization 'GE Oil & Gas' -> 'General Electric Company' on 7 enrollments, Updated organization 'GE Oil & Gas' -> 'General Electric Company' on 1 archived enrollments, Deleted organization 'GE Oil & Gas' (id=43983), Updated organization 'GE' -> 'General Electric Company' on 427 enrollments, Updated organization 'GE' -> 'General Electric Company' on 62 archived enrollments, Deleted organization 'GE' (id=27823), Deleted organization 'Harman/Becker Automotive Systems GmbH' (id=75365), Updated organization 'Hewlett Packard Enterprise' -> 'Hewlett Packard Enterprise Development LP' on 847 enrollments, Updated organization 'Hewlett Packard Enterprise' -> 'Hewlett Packard Enterprise Development LP' on 3796 archived enrollments, Deleted organization 'Hewlett Packard Enterprise' (id=38349), Updated organization 'ScaleArc' -> 'Ignite Enterprise Software Solutions, Inc.' on 8 enrollments, Updated organization 'ScaleArc' -> 'Ignite Enterprise Software Solutions, Inc.' on 1 archived enrollments, Deleted organization 'ScaleArc' (id=40647), Updated organization 'IMC Pacific' -> 'IMC Pacific Pty Ltd' on 7 enrollments, Updated organization 'IMC Pacific' -> 'IMC Pacific Pty Ltd' on 2 archived enrollments, Deleted organization 'IMC Pacific' (id=61433), Deleted organization 'Infront' (id=74253), Updated organization 'JD.com Inc.' -> 'JD.com, Inc.' on 15 enrollments, Updated organization 'JD.com Inc.' -> 'JD.com, Inc.' on 13 archived enrollments, Deleted organization 'JD.com Inc.' (id=59304), Deleted organization 'Kontron S&T' (id=75373), Added organization 'Labn Consulting' (id=78067), Deleted organization 'LabN' (id=75908), Deleted organization 'LAIKA LLC' (id=75375), Updated organization 'Legrand' -> 'Legrand North America' on 8 enrollments, Deleted organization 'Legrand' (id=47313), Updated organization 'Luxoft for UBS' -> 'Luxoft Global Operations GmbH' on 12 enrollments, Deleted organization 'Luxoft for UBS' (id=33737), Updated organization 'Aquantia' -> 'Marvell International Ltd' on 52 enrollments, Updated organization 'Aquantia' -> 'Marvell International Ltd' on 20 archived enrollments, Deleted organization 'Aquantia' (id=26707), Updated organization 'Marvell Semiconductor' -> 'Marvell International Ltd' on 657 enrollments, Updated organization 'Marvell Semiconductor' -> 'Marvell International Ltd' on 68 archived enrollments, Deleted organization 'Marvell Semiconductor' (id=27786), Updated organization 'MayaData Inc.' -> 'MayaData Inc. (f/k/a CloudByte, Inc)' on 1 enrollments, Updated organization 'MayaData Inc.' -> 'MayaData Inc. (f/k/a CloudByte, Inc)' on 26 archived enrollments, Deleted organization 'MayaData Inc.' (id=74221), Updated organization 'Mercedes-Benz' -> 'Mercedes-Benz AG' on 57 enrollments, Updated organization 'Mercedes-Benz' -> 'Mercedes-Benz AG' on 346 archived enrollments, Deleted organization 'Mercedes-Benz' (id=33388), Deleted organization 'Metamarkets Group Inc' (id=75382), Updated organization 'MongoDB' -> 'monday.com Limited' on 132 enrollments, Updated organization 'MongoDB' -> 'monday.com Limited' on 176 archived enrollments, Deleted organization 'MongoDB' (id=33481), Updated organization 'Naver' -> 'Naver Corporation' on 118 enrollments, Updated organization 'Naver' -> 'Naver Corporation' on 126 archived enrollments, Deleted organization 'Naver' (id=26595), Updated organization 'Altemista' -> 'NTT DATA EMEA Ltd.' on 14 enrollments, Deleted organization 'Altemista' (id=32327), Updated organization 'OctoPerf ' -> 'Oblivion Cloud Control Pvt Ltd' on 10 enrollments, Deleted organization 'OctoPerf ' (id=47115), Updated organization 'jQuery UI' -> 'OpenJS Foundation' on 20 enrollments, Updated organization 'jQuery UI' -> 'OpenJS Foundation' on 1320 archived enrollments, Deleted organization 'jQuery UI' (id=46338), Updated organization 'Responsys' -> 'Oracle America Inc.' on 11 enrollments, Deleted organization 'Responsys' (id=33370), Updated organization 'Oracle India' -> 'Oracle India Pvt. Ltd.' on 2 enrollments, Deleted organization 'Oracle India' (id=72449), Deleted organization 'Overclock Labs, Inc' (id=74625), Updated organization 'OVH Cloud' -> 'OVH SAS' on 10 enrollments, Updated organization 'OVH Cloud' -> 'OVH SAS' on 25 archived enrollments, Deleted organization 'OVH Cloud' (id=41349), Updated organization 'Palm' -> 'Palm Ventures Group, Inc.' on 15 enrollments, Updated organization 'Palm' -> 'Palm Ventures Group, Inc.' on 1425 archived enrollments, Deleted organization 'Palm' (id=55022), Added organization 'Plainsight' (id=78068), Updated organization 'Sixgill, LLC' -> 'Plainsight' on 8 enrollments, Deleted organization 'Sixgill, LLC' (id=57292), Updated organization 'Grafana Labs' -> 'Raintank, Inc. (Grafana Labs)' on 469 enrollments, Updated organization 'Grafana Labs' -> 'Raintank, Inc. (Grafana Labs)' on 2455 archived enrollments, Deleted organization 'Grafana Labs' (id=28654), Updated organization 'Redspread' -> 'Red Hat, Inc.' on 24 enrollments, Updated organization 'Redspread' -> 'Red Hat, Inc.' on 4 archived enrollments, Deleted organization 'Redspread' (id=28976), Updated organization 'SAP Hybris' -> 'SAP' on 5 enrollments, Updated organization 'SAP Hybris' -> 'SAP' on 5 archived enrollments, Deleted organization 'SAP Hybris' (id=51642), Deleted organization 'SAP America Inc' (id=75407), Updated organization 'Sky.' -> 'Sky UK Limited' on 1 enrollments, Updated organization 'Sky.' -> 'Sky UK Limited' on 6 archived enrollments, Deleted organization 'Sky.' (id=70362), Added organization 'Smartsheet, Inc.' (id=78069), Updated organization 'Smartsheet.com' -> 'Smartsheet, Inc.' on 33 enrollments, Updated organization 'Smartsheet.com' -> 'Smartsheet, Inc.' on 68 archived enrollments, Deleted organization 'Smartsheet.com' (id=52531), Updated organization 'Stream' -> 'Stream.IO, Inc' on 20 enrollments, Updated organization 'Stream' -> 'Stream.IO, Inc' on 76 archived enrollments, Deleted organization 'Stream' (id=35280), Updated organization 'Tableau' -> 'TABLEAU SOFTWARE, LLC' on 44 enrollments, Deleted organization 'Tableau' (id=28414), Deleted organization 'SolaryCity' (id=75412), Updated organization 'Tieto Czech s.r.o' -> 'TietoEVRY' on 30 enrollments, Updated organization 'Tieto Czech s.r.o' -> 'TietoEVRY' on 201 archived enrollments, Deleted organization 'Tieto Czech s.r.o' (id=39870), Deleted organization 'Works Applications Co. Ltd' (id=75444), Updated organization 'Yuneec USA, Inc.' -> 'Yuneec, Advanced Technology Labs AG' on 6 enrollments, Updated organization 'Yuneec USA, Inc.' -> 'Yuneec, Advanced Technology Labs AG' on 8 archived enrollments, Deleted organization 'Yuneec USA, Inc.' (id=57479), Updated organization 'ZoomInfo' -> 'ZoomInfo Technologies LLC' on 1 enrollments, Deleted organization 'ZoomInfo' (id=69707), Organizations: added:5 renamed:0 deleted:66 skipped:0, Enrollments: conflicts:15 archive conflicts:0 updated:28362"}
```
2nd run:
```
{"text":"Added organization 'Deutsche Niederlassung der FRIDAY Insurance S.A.' (id=78070), Deleted organization 'Deutsche Niederlassung der FRIDAY Insurance S.A.' (id=78070), Added organization 'JD.com Inc.' (id=78071), Deleted organization 'JD.com Inc.' (id=78071), Added organization 'Tieto Czech s.r.o' (id=78072), Deleted organization 'Tieto Czech s.r.o' (id=78072), Organizations: added:3 renamed:0 deleted:3 skipped:0, Enrollments: conflicts:0 archive conflicts:0 updated:0"}
```
So we have 3 circular orgs:
- Deutsche Niederlassung der FRIDAY Insurance S.A.
- JD.com Inc.
- Tieto Czech s.r.o

cc @bumblebee89

**Comment by bumblebee89** [comment]
@lukaszgryglicki  thanks. Corrections for test in this [PR](https://github.com/LF-Engineering/dev-analytics-affiliation/pull/107)

and a PR on prod with changes https://github.com/LF-Engineering/dev-analytics-affiliation/pull/107

**Comment by lukaszgryglicki** [comment]
On it.

**Pull request merged**
